nncf/tensorflow/sparsity/rb/algorithm.py

```diff
@@ -190,3 +191,6 @@ def compression_rate(self, compression_rate: float) -> None:
 
     def disable_scheduler(self):
         self._scheduler = StubCompressionScheduler()
+
+    def compression_stage(self) -> CompressionStage:
```

Comment:
Could you briefly explain why you implemented `compression_stage` in `RBSparsityController` this way?

Reply:
compression_stage was not abstract method, but it should be
And two controllers does not have implementation for this method, as result `compression_stage` returns `None` for them.
- `NoCompressionAlgorithmController` - added return CompressionStage.UNCOMPRESSED
- `RBSparsityController` - keep return None as on develop (looks like there should be a logic to return stage that depends from sparsity_level)

Reply:
Is it possible to throw a runtime error and not return None?

Reply:
It breaks tests https://github.com/openvinotoolkit/nncf/actions/runs/12651525603/job/35252297218
Created jira ticket 160174